fmf.spec

```diff
@@ -1,16 +1,19 @@
-Name: fmf
-Version: 1.3.0
-Release: 1%{?dist}
+Name:          fmf
+Version:       0.0.0
+Release:       1%{?dist}
```

Comment:
I've dropped the `%autorelease` here because it doesn't really make sense without `%autochangelog` as well, and probably that needs to be done in a separate PR